src/components/ebay-video/index.marko

```diff
@@ -66,4 +66,8 @@ static var ignoredAttributes = [
             <ebay-progress-spinner a11y-text=(input.a11yLoadText || "Loading")/>
         </else>
     </div>
+    <div style="display:none">
```

Comment:
Is this supposed to be inlined? Since it's hidden and I'm not seeing anything that would hook into it and change the display, what's the reason why it's there?